test/utils/runners.go

```diff
@@ -1537,6 +1540,11 @@ func attachConfigMaps(template *v1.PodTemplateSpec, configMapNames []string) {
 	template.Spec.Containers[0].VolumeMounts = mounts
 }
 
+func (config *RCConfig) getTerminationGracePeriodSeconds() *int64 {
+	x := int64(math.Max(float64(config.TerminationGracePeriodSeconds), 1))
```

Comment:
What if someone sets it explicitly to 0?

Reply:
Fixed